auto_py_to_exe/web/index.html

```diff
@@ -66,6 +66,11 @@
                         </label>
                         <select id="language-selection"></select>
                     </span>
+                    <span id="theme">
+                        <small id="head-theme-label">Theme: </small>
+                        <span id="on-dark-theme-label" style="display: inline;">Light</span>
+                        <span id="on-light-theme-label" style="display: none;">Dark</span>
+                    </span>
```

Comment:
> Well, how about translation of label (theme, light, dark keyword)?

Good point, adding translations ourselves from Google Translate would be ok, as others can correct them if they're wrong.

However considering translations, maybe the icon might be better. How about something like this:

![image](https://github.com/brentvollebregt/auto-py-to-exe/assets/19944838/0a4ed00a-27c5-432a-ae29-6f555ed979c0)

Positioned above the other links, and more importantly, the same height as them (20px).

Reply:
That's exactly my point!